patches/net/minecraft/client/sounds/MusicManager.java.patch

```diff
@@ -0,0 +1,25 @@
+--- a/net/minecraft/client/sounds/MusicManager.java
++++ b/net/minecraft/client/sounds/MusicManager.java
+@@ -25,6 +_,13 @@
+ 
+     public void tick() {
+         Music music = this.minecraft.getSituationalMusic();
++        Music posted = net.neoforged.neoforge.client.ClientHooks.selectMusic(music);
```

Comment:
Just reassign `music` here instead of making a second local, it avoids needing to patch `this.startPlaying` further down.